api/app/assets/javascripts/api/v1/assignments.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Why are we generating .coffee files ? Are there plans to use this ? I would prefer these to be dropped these if not.

Reply:
These were generated when I made the models in rails. Do you have a list of files that should be removed if I generate model through the rails command? Or is it preferred that I make these files manually?

Reply:
I've removed the non-controller files. Does it look okay now?

Reply:
This looks good to me !